packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -3468,7 +3547,8 @@ export class ThinEngine extends AbstractEngine {
         width: number,
         height: number,
         samples = 1,
-        depthTextureFormat?: number
+        depthTextureFormat?: number,
+        dontBindRenderBufferToFrameBuffer = false

```

Comment:
Not sure to understand when this would be used ? (creating the buffers but not binding them)

Reply:
It's used in `engine._createInternalTexture`, when creating the MSAA texture (=`WebGLRenderBuffer`). In that case, the MSAA framebuffer does not exist yet (this framebuffer is managed by `WebGLRenderTargetWrapper`), so we can't bind the render buffer to it. Binding the MSAA render buffer is managed by the new code in `WebGLRenderTargetWrapper._bindTextureRenderTarget` (at the end of the method).